src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToTargetEolFrameworks.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToTargetEolFrameworks : SdkTest
+    {
+        public GivenThatWeWantToTargetEolFrameworks(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Theory]
+        [InlineData("netcoreapp1.0")]
+        [InlineData("netcoreapp1.1")]
+        [InlineData("netcoreapp2.0")]
+        [InlineData("netcoreapp2.2")]
+        [InlineData("netcoreapp3.0")]
+        public void It_warns_that_framework_is_out_of_support(string targetFrameworks)
```

Comment:
The targetFrameworks parameter isn't actually being used in the test, except to change the name of the project.  Did you intend to use it to set the TestProject TargetFrameworks on line 31?

Reply:
I'm fixing these up.